### PR TITLE
Feature/exchange

### PR DESCRIPTION
**Pull request by MarcusSchweizer91**

### CODE REVIEW

**Review by MoinMarcell — APPROVED** [review]
LGTM but next time give attention to formation, many empty lines :)

File: frontend/src/MovieGallery/component/MovieExchangeCard.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+
```
Comment (devBK84) [inline frontend/src/MovieGallery/component/MovieExchangeCard.css:2]: Delete line

File: frontend/src/MovieGallery/component/MovieExchangeCard.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            price:price
+        }
+        props.editMovie(newMovie)
+        setShowButton(false)
+    }
+
+
+
+
+
```
Comment (devBK84) [inline frontend/src/MovieGallery/component/MovieExchangeCard.tsx:49]: Delete line

File: frontend/src/MovieGallery/model/MovieToExchange.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+
```
Comment (devBK84) [inline frontend/src/MovieGallery/model/MovieToExchange.ts:2]: Delete line

**Review by devBK84 — APPROVED** [review]
GJ

**Pull request merged**